query/src/test/java/org/infinispan/query/api/InfinispanDirectoryNonIndexedValuesTest.java

```diff
@@ -21,10 +21,7 @@ protected EmbeddedCacheManager createCacheManager() throws Exception {
             .enable()
             .addIndexedEntity(TestEntity.class)
             .addIndexedEntity(AnotherTestEntity.class)
-            .addProperty("default.indexmanager", "near-real-time")
-            .addProperty("default.directory_provider", "local-heap")
-            .addProperty("error_handler", StaticTestingErrorHandler.class.getName())
-            .addProperty("lucene_version", "LUCENE_CURRENT");
+            .addProperty(SearchConfig.DIRECTORY_TYPE, SearchConfig.HEAP);
```

Comment:
What happened to near-real-time and the error handler? Regarding the latter, many tests use it currently

Reply:
And I'd rather keep some tests using near real time rather than have 100% using local-heap

Reply:
near-real-time is enabled in Search 6 by setting the property `hibernate.search.backends.<backend name>.index_defaults.io.refresh_interval` to something higher than 0.
See https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/#backend-lucene-io-refresh.
About error handler I restored it for the tests, the new name in Search 6 is failure handler. So now `StaticTestingErrorHandler` extends `FailureHandler`.

Reply:
From ISPN the prefix `hibernate.search.backends.<backend name>` is added automatically to all the properties provided to Search that do not start with ``hibernate.search.`. That is the behavior at the moment, of course we can change it.